src/js/graphql_endpoint/server.js

```diff
@@ -57,10 +57,24 @@ const middleware = [cors(corsDelegate), validateJwt];
 
 app.options('*', cors(corsDelegate));
 
-app.use('/graphQlEndpoint/graphql', middleware, graphqlHTTP({
-    schema: schema,
-    graphiql: IS_LOCAL
-}));
+// app.use('/graphQlEndpoint/graphql', middleware, graphqlHTTP({
+//     schema: schema,
+//     graphiql: IS_LOCAL
+// }));
+
+
+app.use('/graphQlEndpoint/graphql', middleware, graphqlHTTP(async (request, response, graphQLParams) => {
+        console.debug({request: request, response: response, graphQLParams: graphQLParams});
```

Comment:
Let's make sure we remove this log later, since it will contain cookies from the user that we wouldn't want to accidentally log.